src/PIL/TiffImagePlugin.py

```diff
@@ -991,9 +991,11 @@ def tobytes(self, offset: int = 0) -> bytes:
         if stripoffsets is not None:
             tag, typ, count, value, data = entries[stripoffsets]
             if data:
-                msg = "multistrip support not yet implemented"
-                raise NotImplementedError(msg)
-            value = self._pack("L", self._unpack("L", value)[0] + offset)
+                size, handler = self._load_dispatch[typ]
+                values = [val + offset for val in handler(self, data, self.legacy_api)]
```

Comment:
```suggestion
                values = [val + offset for val in handler(self, data)]
```
Throwing this suggestion out there, see what happens - considering that this should be [SHORT or LONG](https://web.archive.org/web/20230619074139/https://www.awaresystems.be/imaging/tiff/tifftags/stripoffsets.html) (or LONG8), it doesn't matter whether the legacy API is in use or not, as they're all passed to https://github.com/python-pillow/Pillow/blob/b557876ec3aa4d5d236d8044519f1c613031fbee/src/PIL/TiffImagePlugin.py#L502-L505

Reply:
I think it's more correct to pass the value since we have it. The value not being used is an implementation detail that this call shouldn't be concerned with.